src/main/java/org/opensearch/knn/index/mapper/CompressionLevel.java

```diff
@@ -29,6 +29,8 @@ public enum CompressionLevel {
     x16(16, "16x", new RescoreContext(3.0f, false), Set.of(Mode.ON_DISK)),
     x32(32, "32x", new RescoreContext(3.0f, false), Set.of(Mode.ON_DISK));
 
+    public static final CompressionLevel MAX_COMPRESSION_LEVEL = CompressionLevel.x32;
```

Comment:
should we have 64 as a max compression level? I don't have a solid point to have 64 but I think have 1 more extra compression is always good.

Reply:
I think that makes sense. I guess for 64x, default for all dimensions should probably be 5x.